radar/2023-08-01/bdd.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "BDD with Gherkin, Cucumber, and Cypress"
+ring: adopt
+quadrant: techniques
+featured: true
+---
+
+Behavior-Driven Development (BDD) is a collaborative approach to software development that bridges the communication gap between business and IT. BDD helps teams focus on delivering features that truly matter by emphasizing the behavior of an application from the end user's perspective.
+
+[Gherkin](https://cucumber.io/docs/gherkin/) is Cucumber's language to define test cases. It's designed to be easy to read and lets you write scripts in a natural language style.
+
+[Cucumber](https://cucumber.io/) is a tool that supports BDD. It reads Gherkin documents and validates that the code performs as expected.
+
+[Cypress](https://www.cypress.io/) is an end-to-end testing framework that can be used alongside Gherkin and Cucumber for testing web applications. It's popular for its easy setup and fast, reliable testing capabilities. Cypress can validate various aspects of your application, ranging from the UI functionality to complete workflows.
```

Comment:
Cypress is already on our tech radar: https://infonl.github.io/info-techradar/tools/cypress.html I would just link to that instead of repeating it here?

Reply:
will link it

Reply:
and now also good to mention Playwright in this context I guess?